frontend/src/components/Dropdown/components/SelectCombobox/ComboboxClearButton.tsx

```diff
@@ -26,7 +27,7 @@ export const ComboboxClearButton = (): JSX.Element | null => {
       aria-label={clearButtonLabel}
       onClick={handleClearSelection}
       __css={styles.clearbutton}
-      color={inputValue ? 'secondary.500' : undefined}
+      color={inputValue || selectedItem ? 'secondary.500' : undefined}
```

Comment:
how about
```suggestion
      {...(inputValue ? { color: 'secondary.500' } : {})}
```

instead of passing in `undefined` to `color` prop

Reply:
No difference and `undefined` makes it easier to read imho